poncho/src/poncho/package_serverize.py

```diff
@@ -50,7 +50,9 @@ def remote_wrapper(event, q=None):
 
 def send_configuration(config):
     config_string = json.dumps(config)
-    print(len(config_string) + 1, "\\n", config_string, flush=True)
+    config_cmd = f"{len(config_string) + 1}\\n{config_string}\\n"
```

Comment:
I'm not sure you need the double slash. I think it ends up working because `write` interprets the result "\n". I'd suggest we use a single slash here so we don't get confused in the future.

Reply:
I just tested it, and the double slash is required because there is a level of indirection here. The python code in package_serverize is stored as a string, so if it has single slashes like
`config_cmd = f"{len(config_string) + 1}\n{config_string}\n"`
when the serverless function is created, the \n are treated as literal newlines, and we get this:
`config_cmd = f"{len(config_string) + 1}
{config_string}
"`